src/plugins/news/screens/article/schemas.js

```diff
@@ -2,6 +2,9 @@ import ArticleV1Mixin from '@triniti/schemas/triniti/news/mixin/article/ArticleV
 import SearchArticlesRequestV1Mixin from '@triniti/schemas/triniti/news/mixin/search-articles-request/SearchArticlesRequestV1Mixin';
 import resolveSchema from '@triniti/cms/utils/resolveSchema';
 
+import NodeLabelsUpdatedV1 from '@gdbots/schemas/gdbots/ncr/event/NodeLabelsUpdatedV1';
```

Comment:
@rsumilang because these are concrete the schemas ref and there to solve the "lookup" problem we don't need it.